src/dom/dom.js

```diff
@@ -1318,36 +2015,49 @@ p5.prototype.createVideo = function (src, callback) {
 /** AUDIO STUFF **/
 
 /**
- * Creates a hidden HTML5 `&lt;audio&gt;` element in the DOM for simple audio
- * playback. The first parameter can be either a single string path to a
- * audio file, or an array of string paths to different formats of the same
- * audio. This is useful for ensuring that your audio can play across
- * different browsers, as each supports different formats.
- * See <a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats'>this
- * page for further information about supported formats</a>.
+ * Creates a hidden `&lt;audio&gt;` element for simple audio playback.
+ * Returns a new <a href="#/p5.MediaElement">p5.MediaElement</a> object.
+ *
+ * The first parameter, `src`, is the path the video. If a single string is
+ * passed, as in `'assets/video.mp4'`, a single video is loaded. An array
+ * of strings can be used to load the same video in different formats. For
+ * example, `['assets/video.mp4', 'assets/video.ogv', 'assets/video.webm']`.
+ * This is useful for ensuring that the video can play across different
+ * browsers with different capabilities. See
+ * <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats" target="_blank">MDN</a>
+ * for more information about supported formats.
+ *
+ * The second parameter, `callback`, is optional. It's a function to call once
+ * the audio is ready to play.
  *
  * @method createAudio
- * @param  {String|String[]} [src] path to an audio file, or array of paths
- *                             for supporting different browsers
- * @param  {Function} [callback] callback function to be called upon
- *                             'canplaythrough' event fire, that is, when the
- *                             browser can play the media, and estimates that
- *                             enough data has been loaded to play the media
- *                             up to its end without having to stop for
- *                             further buffering of content
- * @return {p5.MediaElement}   pointer to audio <a href="#/p5.MediaElement">p5.MediaElement</a>
+ * @param  {String|String[]} [src] path to an audio file, or an array of paths
+ *                                 for supporting different browsers.
+ * @param  {Function} [callback]   function to call once the audio is ready to play.
+ * @return {p5.MediaElement}       new <a href="#/p5.MediaElement">p5.MediaElement</a> object.
  * @example
- * <div><code>
- * let ele;
+ * <div>
+ * <code>
+ * let beat;
+ *
  * function setup() {
- *   ele = createAudio('assets/beat.mp3');
+ *   background(200);
+ *   text('Double-click to stop', 25, 55);
+ *
+ *   // Load the audio.
+ *   beat = createAudio('assets/beat.mp3', () => {
+ *     // Play the audio in a loop once it's loaded.
+ *     beat.loop();
```

Comment:
This one also doesn't work for me if I go directly to the URL for this doc. Maybe we can start it on click, or only load the audio on click? Probably worth mentioning for both audio and video that this might happen if there hasn't been a user interaction